### PR TITLE
feat: add homepage hero section

### PR DESCRIPTION
**Pull request by blp100**
## Description

figma:
desktop: https://www.figma.com/design/6JPbEOj0WIGCAr3gJQU0Cb/%E5%85%AD%E8%A7%92_ChillKa?node-id=108-392&t=oYbzzMUPLPczJ7Ll-4
mobile: 
https://www.figma.com/design/6JPbEOj0WIGCAr3gJQU0Cb/%E5%85%AD%E8%A7%92_ChillKa?node-id=285-5480&t=oYbzzMUPLPczJ7Ll-4

## Test

https://github.com/ChillKa/chillka-frontend/assets/70035368/94a4ffc5-d58b-4fbb-a061-e782581b6685

### CODE REVIEW

File: src/components/HeroSection.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+};
+
+const HeroSection = ({ className }: HeroSectionProps) => {
+  return (
+    <section
+      className={cn(
+        'relative flex max-w-[81rem] flex-grow flex-col gap-10 px-3 pb-[17.5rem] pt-20 xl:items-center xl:p-0 xl:py-[7.625rem]',
+        className
+      )}
+    >
```
Comment (ellallu0903) [inline src/components/HeroSection.tsx:16]: 調整手機版的間距
```suggestion
    <section
      className={cn(
        'relative flex max-w-[81rem] flex-grow flex-col gap-6 px-3 pb-[17.5rem] pt-20 xl:items-center xl:gap-10 xl:p-0 xl:py-[7.625rem]',
        className
      )}
    >
```

File: src/components/HeroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <Image
+        src="https://images.unsplash.com/photo-1622495893726-f1478dcebc03?q=100&w=1296&h=fit&fm=webp"
+        fill
+        alt="Hero Image"
+        className="left-0 top-0 -z-10 h-full w-full object-cover object-[10%_50%] xl:object-center"
+      />
```
Comment (ellallu0903) [inline src/components/HeroSection.tsx:22]: 照片上面有一層濾鏡效果
1. `Image` 改成 `-z-20`
2. 新增一個 div 做效果
```suggestion
      <Image
        src="https://images.unsplash.com/photo-1622495893726-f1478dcebc03?q=100&w=1296&h=fit&fm=webp"
        fill
        alt="Hero Image"
        className="left-0 top-0 -z-20 h-full w-full object-cover object-[10%_50%] xl:object-center"
      />
      <div className="absolute inset-0 -z-10 bg-black bg-opacity-30" />
```

File: src/components/HeroSection.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          fill="#ffffff"
+        />
+      </svg>
+      <H2 className="text-white">無聊想找伴或參加活動嗎</H2>
```
Comment (ellallu0903) [inline src/components/HeroSection.tsx:37]: 1. 內容少了一個問號
2. 待調整：手機版是 `H4`，PC 版是 `H2`
```suggestion
      <H2 className="text-white">無聊想找伴或參加活動嗎？</H2>
```

File: src/components/HeroSection.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      <h1 className="text-[4rem]/[4.8rem] font-thin -tracking-[0.012em] text-white xl:text-center xl:text-[7.5rem]/[9rem]">
```
Comment (ellallu0903) [inline src/components/HeroSection.tsx:38]: 設計稿字體 `font-weight: 250`，最接近的應該是 `font-extralight (font-weight: 200;)`
```suggestion
      <h1 className="text-[4rem]/[4.8rem] font-extralight -tracking-[0.012em] text-white xl:text-center xl:text-[7.5rem]/[9rem]">
```

**Review by ellallu0903 — COMMENTED** [review]
1. 調整樣式與內容
2. 待補：Scroll down
image

[PC 版](https://www.figma.com/design/6JPbEOj0WIGCAr3gJQU0Cb/%E5%85%AD%E8%A7%92_ChillKa?node-id=107-201&t=Ac5kiHp8fBa8DcUd-0)
[Mobile 版](https://www.figma.com/design/6JPbEOj0WIGCAr3gJQU0Cb/%E5%85%AD%E8%A7%92_ChillKa?node-id=277-3117&t=F7nF9YEHZStmQMQN-0)

File: src/components/HeroSection.tsx (hunk 1 above)
Comment (blp100) [reply src/components/HeroSection.tsx]: fix on this commit: 75fb3d68c9678ba99d362222c9e74bb5b333bd61

File: src/components/HeroSection.tsx (hunk 2 above)
Comment (blp100) [reply src/components/HeroSection.tsx]: fix on this commit: 75fb3d68c9678ba99d362222c9e74bb5b333bd61

File: src/components/HeroSection.tsx (hunk 3 above)
Comment (blp100) [reply src/components/HeroSection.tsx]: fix on this commit: 75fb3d68c9678ba99d362222c9e74bb5b333bd61

**Review by ellallu0903 — APPROVED** [review]

**Pull request merged**